backend/src/apiserver/storage/object_store.go

```diff
@@ -143,6 +144,27 @@ func (m *MinioObjectStore) GetSignedUrl(bucketConfig *objectstore.Config, secret
 		return "", err
 	}
 	reqParams := make(url.Values)
+	reqParams.Set("response-content-disposition", "attachment; filename=\""+key+"\"")
```

Comment:
If you go with my comment of having the method `GetSignedUrlWithQueryParams(bucketConfig *objectstore.Config, secret *v1.Secret, expirySeconds time.Duration, artifactURI string, queryParams url.Values) `, then you can just have `GetSignedUrl` wrap `GetSignedUrlWithQueryParams` and pass the content disposition to `queryParams`.

This will reduce code duplication.